academic-services/graph/reference-data-schema.md

```diff
@@ -57,6 +68,11 @@ Column # | Name | Type | Note
 
 ## ConferenceInstances.txt
```

Comment:
Can you standardize the format for all of these by removing .txt and adding spaces between words?